website/src/layouts/StarlightSplashLayout.astro

```diff
@@ -0,0 +1,76 @@
+---
+// @ts-expect-error
+import type { LocaleData } from "@astrojs/starlight/utils/slugs";
+import Page from "@astrojs/starlight/components/Page.astro";
+
+export interface Props {
```

Comment:
@Yan-Thomas I think we could use this layout for Playground page too
Maybe we just need to hide the page title, do you think it's possible?

Reply:
Yes, that's what I thought when I saw the website rework issue, should be straightforward in theory. 

I'd say to keep this layout like so and then tackle a refactor in the Playground PR so we abstract it properly, as there might be a few edge cases to consider and as I also am not familiarized with the current Playground implementation.